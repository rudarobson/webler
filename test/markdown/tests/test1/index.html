<!DOCTYPE html>
<html>

<body>
  <!-- markdown: -->
  ```html
  <div> this is a markdown </div>
  ```
  <!-- /markdown -->
</body>

</html>
