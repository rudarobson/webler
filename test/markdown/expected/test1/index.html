<!DOCTYPE html>
<html>

<body>
  <pre><code class="lang-html">  &lt;div&gt; this is a markdown &lt;/div&gt;
</code></pre>

</body>

</html>
